file-formats/enhs/type/zif_seef_aff_enhs_v1.intf.abap

```diff
@@ -0,0 +1,223 @@
+      "! <p class="shorttext">Description of the Filter</p>
+      "! Description of the filter
+      "! $required
+      description          TYPE c LENGTH 255,
+      "! <p class="shorttext">Type of the Filter</p>
+      "! Type of the filter
+      "! $required
+      type                 TYPE ty_filter_type,
+      "! <p class="shorttext">Only constant Filter Values</p>
+      "! The filter has only constant filter values
+      only_constant_values TYPE abap_bool,
+      "! <p class="shorttext">Value Check Ddic</p>
+      "! Value check via domain or data element
+      value_check_ddic     TYPE ty_value_check_ddic,
+      "! <p class="shorttext">Value Check Class</p>
+      "! Value check via class
+      value_check_class    TYPE ty_value_check_class,
+    END OF ty_badi_def_filter.
+
+  TYPES ty_badi_def_filters TYPE STANDARD TABLE OF ty_badi_def_filter WITH DEFAULT KEY.
+  TYPES ty_example_classes TYPE STANDARD TABLE OF zif_aff_types_v1=>ty_object_name_30 WITH DEFAULT KEY.
+
+
+  TYPES:
+    "! <p class="shorttext">BAdI Definition</p>
+    "! Information about this BAdI definition
+    BEGIN OF ty_badi_def,
+      "! <p class="shorttext">Name of the BAdI Definition</p>
+      "! Name of the BAdI definition
+      "! $required
+      name               TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Description of the BAdI Definition</p>
+      "! Description of the BAdI definition
+      "! $required
+      description        TYPE c LENGTH 255,
+      "! <p class="shorttext">Interface of the BAdI Definition</p>
+      "! Interface of the BAdI definition
+      "! $required
+      interface          TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Instantiation of the BAdI Definition</p>
+      "! Instantiation of the BAdI definition
+      instantiation      TYPE ty_instantiation,
+      "! <p class="shorttext">Single- or Multiple Use BAdI</p>
+      "! Indicator whether the BAdI definition is single- or multiple usable
+      multiple_use       TYPE abap_bool,
+      "! <p class="shorttext">SAP Internal</p>
+      "! SAP internal
+      sap_internal       TYPE abap_bool,
+      "! <p class="shorttext">Example Classes of the BAdI Definition</p>
+      "! Example classes of the BAdI definition
+      example_classes    TYPE ty_example_classes,
+      "! <p class="shorttext">Use Fallback Class</p>
+      "! Use fallback class in case no implementation is executed
+      use_fallback_class TYPE abap_bool,
+      "! <p class="shorttext">Name of the Default/Fallback Class</p>
+      "! Name of the default/fallback class
+      fallback_class     TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Limited Filter Use</p>
+      "! Limited filter use
+      filter_limitation  TYPE abap_bool,
+      "! <p class="shorttext">Documentation Id</p>
+      "! Documentation Id
+      documentation_id   TYPE c LENGTH 60,
```

Comment:
Shouldn't this be 40 characters?

Reply:
The types that are actually used internally have 60 characters

Reply:
OK, Thanks